content/docs/reference-react-dom.md

```diff
@@ -6,25 +6,25 @@ category: Reference
 permalink: docs/react-dom.html
 ---
 
-If you load React from a `<script>` tag, these top-level APIs are available on the `ReactDOM` global. If you use ES6 with npm, you can write `import ReactDOM from 'react-dom'`. If you use ES5 with npm, you can write `var ReactDOM = require('react-dom')`.
+如果使用 `<script>` 標籤載入 React 這些頂層 API 就可以在全域 `ReactDOM` 使用。如果你使用 ES6 和 npm，你可以寫成 `import ReactDOM from 'react-dom'`。如果你使用 ES5 和 npm，你可以寫成 `var ReactDOM = require('react-dom')`。
 
-## Overview {#overview}
+## 概述 {#overview}
```

Comment:
> Hi @abrcdf1023, thank you for your translation. I think before translate you should check our [Glossary](https://github.com/reactjs/zh-hant.reactjs.org/wiki/Glossary) for terms translation, e.g. `render` do not translate `渲染`.
> 
> cc @chloewlin for double check my reviewing, thank you.

Hi, I read Glossary already, and I translated `render` into `渲染` is because I think `render` should be translate only if it be described as a function.
I don't know if we should keep the term all the time because people may misunderstand it.